source/administration/devops-command-center.rst

```diff
@@ -0,0 +1,284 @@
+in the team they are created in.
+
+Private incidents
+~~~~~~~~~~~~~~~~~~~~~
+
+Private incidents are invite-only. Participants can be added by inviting them to the incident's primary channel. Like private channels, private incidents do not appear on
+the incident list of users who are not participants.
+
+To view incidents associated with your Mattermost team, select the **Incidents** icon in the channel header to open the RHS menu. If there are no active incidents, you can
+start one or you can view incident logs.
+
+Incidents Backstage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To open the Incidents backstage Select **Incidents & Playbooks** from the Main Menu , which lists all incidents associated with the current team. This main menu item is available to all Mattermost users.
+
+All incidents for the current team are listed for review with the following details for each incident:
+
+- Name
+- Status (**Ongoing** or **Ended**)
+- Start Date
+- End Date, if ended, otherwise **--**
+- Commander
+
+All dates are rendered as per the locale and timezone of the active user.
+
+Viewing incident details
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To view details of active incidents, select **Incidents** from the plugin menu to open the RHS panel.
+
+Listed incidents include the following information:
+
+- The current Commander’s profile picture and username
+- Zero or more checklist items
+- The channels associated with the incident
+- A button to end the incident
+
+If the incident is active, but in a private channel, you won't be able to see the incident channel name or select **End Incident**. However, the Commander name and incident
+checlist will be visible. If you're a participant in an incident channel, you can modify incident details from within that channel.
+
+You can view all incidents in the incident Backstage via **Main Menu > Incidents & Playbooks**.
+
+Select an incident to view:
+
+- Incident name
+- A link icon to open the corresponding incident channel
+- Status (**Ongoing** or **Ended**)
+- The Commander, including profile picture and username
+- A prompt to export the incident channel
+- The total duration
+- The number of members involved in the incident
+- The number of messages posted to the channel, including another prompt to open the corresponding incident channel
+
+The **Duration** widget displays the duration of the incident. While the incident is ongoing, the end time is displayed as **Ongoing**. When the incident has ended, it
+shows the end time (in the user's timezone). The **Members Involved** widget indicates the total number of users that participated in the channel, either
+by posting a message, being assigned as Commander, or interacting with a checklist. This number is not affected by users leaving the channel, or users joining the channel but not participating.
```

Comment:
This is not true right now, but it will be soon(ish).

Reply:
How soon-ish? :) Should I hold off on this until that release or publish now and then update?